src/actions/pipelineRuns.js

```diff
@@ -50,3 +50,22 @@ export function fetchPipelineRuns({ pipelineName, namespace } = {}) {
     return pipelineRuns;
   };
 }
+
+export function createPipelineRunAction({ payload, namespace }) {
+  return async (dispatch, getState) => {
+    dispatch({ type: 'PIPELINE_RUNS_FETCH_REQUEST' });
+    let headers;
+    try {
+      const requestedNamespace = namespace || getSelectedNamespace(getState());
+      headers = await createPipelineRun(payload, requestedNamespace);
+      const pipelineRuns = await getPipelineRuns(
```

Comment:
I think it returns the PipelineRun and an error: https://godoc.org/github.com/tektoncd/pipeline/pkg/client/clientset/versioned/typed/pipeline/v1alpha1#PipelineRunInterface

However, our dashboard's REST endpoint does not return the PipelineRun

Reply:
Should be able to remove this line now. I'd probably go for doing this in the component instead of in an action but I'll leave that up to you, it would be easier and alot less code. If you do keep it as an action change the action type, this isn't fetching so `PIPELINE_RUNS_FETCH_REQUEST` doesn't seem right.